contrib/get_faas.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+sudo curl -sL https://cli.openfaas.com | sudo sh
```

Comment:
why do `sudo curl`?

Reply:
Should I do it without sudo ? travis doesn't run scripts in root by default so sudo was needed for that

Reply:
You only need `sudo` for the `sh` not for `curl` which doesn't require privileges.